### DISCUSSION TITLE
Welcome to seatreg Discussions!

### DISCUSSION THREAD

**Discussion by SiimKirjanen** [SiimKirjanen/seatreg#33] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by dhirotake** [comment]
Hello.
I am using the [SeatReg] plug-in you developed. This plugin does not have Japanese language, so I translated it. It seems that approval is required after translation, but I don't know how to do it.
Can you help me?

**Reply by SiimKirjanen** [reply]
Hey.
As far as I know those translations need to be approved by someone who has Translation Editor access to Japanese language. I don't have that access. https://make.wordpress.org/polyglots/handbook/translating/glotpress-translate-wordpress-org/
Maybe you can find and contact someone.
There are some names at bottom of the https://translate.wordpress.org/locale/ja/default/wp-plugins/seatreg/ page.

**Reply by dhirotake** [reply]
Thank you for your reply.
I will try to contact you as you told me.
